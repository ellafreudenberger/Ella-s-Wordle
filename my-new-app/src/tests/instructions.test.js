import { render } from '@testing-library/react';
import React from 'react';
import Instructions from './instructions';

test('renders instructions component snapshot', () => {
  const { asFragment } = render(<Instructions />);
  expect(asFragment()).toMatchSnapshot();
});

