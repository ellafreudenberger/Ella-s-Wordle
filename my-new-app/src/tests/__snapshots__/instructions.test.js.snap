// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders instructions component snapshot 1`] = `
<DocumentFragment>
  <div
    class="fade alert alert-success show"
    role="alert"
  >
    <div
      class="alert-heading h4"
    >
      Instructions 
    </div>
    <p>
      Guess the 5 letter word in 6 guesses. If a letter guessed results in a yellow tile, then the letter is in the wrong order of the right word answer. If a letter guessed results in a green tile, then the letter is in the correct order of the right word answer. If a letter guessed results in a red tile, then the letter is not in the right word answer. If all 5 letters guessed result in green tiles, then the right word has been guessed successfully.
    </p>
    <hr />
    <p
      class="mb-0"
    >
      Good luck!
    </p>
  </div>
</DocumentFragment>
`;
